src/SFA.DAS.ProviderApprenticeshipsService.Web/Orchestrators/ApprovedApprenticeshipValidation/ApprovedApprenticeshipValidator.cs

```diff
@@ -1,49 +0,0 @@
-﻿using System;
```

Comment:
This file has been moved to the Validation folder with all the other validators.